src/frequenz/sdk/microgrid/component/_component_states.py

```diff
@@ -39,3 +41,83 @@ def from_pb(
             return cls.UNSPECIFIED
 
         return EVChargerCableState(evc_state)
+
+
+class InverterState(Enum):
+    """Inverter states."""
+
+    UNSPECIFIED = inverter_pb.ComponentState.COMPONENT_STATE_UNSPECIFIED
+    OFF = inverter_pb.ComponentState.COMPONENT_STATE_OFF
+    SWITCHING_ON = inverter_pb.ComponentState.COMPONENT_STATE_SWITCHING_ON
+    SWITCHING_OFF = inverter_pb.ComponentState.COMPONENT_STATE_SWITCHING_OFF
+    STANDBY = inverter_pb.ComponentState.COMPONENT_STATE_STANDBY
+    IDLE = inverter_pb.ComponentState.COMPONENT_STATE_IDLE
+    CHARGING = inverter_pb.ComponentState.COMPONENT_STATE_CHARGING
+    DISCHARGING = inverter_pb.ComponentState.COMPONENT_STATE_DISCHARGING
+    ERROR = inverter_pb.ComponentState.COMPONENT_STATE_ERROR
+    UNAVAILABLE = inverter_pb.ComponentState.COMPONENT_STATE_UNAVAILABLE
+
+    @classmethod
+    def from_pb(
+        cls, inverter_state: inverter_pb.ComponentState.ValueType
+    ) -> InverterState:
+        """Convert a protobuf ComponentState value to InverterState enum.
+
+        Args:
+            inverter_state: protobuf component state to convert.
+
+        Returns:
+            Enum value corresponding to the protobuf message.
+        """
+        return InverterState(inverter_state)
+
+
+class BatteryState(Enum):
+    """Battery states."""
+
+    UNSPECIFIED = battery_pb.ComponentState.COMPONENT_STATE_UNSPECIFIED
+    OFF = battery_pb.ComponentState.COMPONENT_STATE_OFF
+    IDLE = battery_pb.ComponentState.COMPONENT_STATE_IDLE
+    CHARGING = battery_pb.ComponentState.COMPONENT_STATE_CHARGING
+    DISCHARGING = battery_pb.ComponentState.COMPONENT_STATE_DISCHARGING
+    ERROR = battery_pb.ComponentState.COMPONENT_STATE_ERROR
+    LOCKED = battery_pb.ComponentState.COMPONENT_STATE_LOCKED
+
+    @classmethod
+    def from_pb(
+        cls, battery_state: battery_pb.ComponentState.ValueType
+    ) -> BatteryState:
+        """Convert a protobuf ComponentState value to BatteryState enum.
+
+        Args:
+            battery_state: protobuf component state to convert.
+
+        Returns:
+            Enum value corresponding to the protobuf message.
+        """
+        return BatteryState(battery_state)
+
+
+class BatteryRelayState(Enum):
+    """Battery relay states."""
+
+    RELAY_STATE_UNSPECIFIED = battery_pb.RelayState.RELAY_STATE_UNSPECIFIED
+    RELAY_STATE_OPENED = battery_pb.RelayState.RELAY_STATE_OPENED
+    RELAY_STATE_PRECHARGING = battery_pb.RelayState.RELAY_STATE_PRECHARGING
+    RELAY_STATE_CLOSED = battery_pb.RelayState.RELAY_STATE_CLOSED
+    RELAY_STATE_ERROR = battery_pb.RelayState.RELAY_STATE_ERROR
+    RELAY_STATE_LOCKED = battery_pb.RelayState.RELAY_STATE_LOCKED
```

Comment:
here too, I guess the prefix can go away.

Reply:
No enums anymore